hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/contracts/precompile/impl/TransferPrecompile.java

```diff
@@ -288,6 +288,7 @@ protected void extrapolateDetailsFromSyntheticTxn() {
                 impliedTransfersMarshal.assessCustomFeesAndValidate(
                         hbarOnly,
                         0,
+                        0,
```

Comment:
Afaict it's definitely _correct_ to put `0` here, because any unknown alias [will be translated](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/contracts/HederaStackedWorldStateUpdater.java#L123) to `0x00...0000` and then `0.0.0`; leading to `INVALID_ACCOUNT_ID`.

But is that our intention? Or do we want hollow account creation to work inside this precompile as well?